src/commands/Cfacts.js

```diff
@@ -0,0 +1,33 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+class CatFacts extends Command {
+	constructor(...args) {
+		super(...args);
+
+		this.aliases        = ['catfact', 'catfacts']; // meow facts
+		this.module         = 'Fun';
+		this.description    = 'Get random Cat Facts with this commmand';
+		this.usage          = 'catfact';
+		this.example        = 'catfact';
+		this.cooldown       = 5000;
+		this.expectedArgs   = 1;
+	}	
+
+		async execute({ message }) {
+			try {
+				let res = await superagent.get('https://catfact.ninja/fact');
+				
+				return this.sendMessage(message.channel, res.body.value.joke);

```

Comment:
Also `res.body.value.joke` isn't valid. I just tested it, it'd be `res.body.fact`